thefloatingdutchman/level/room/enemy_room.py

```diff
@@ -24,6 +24,9 @@ def update(self, player: PlayerSprite, screen: Surface):
     def get_enemies(self):
         return self._enemy_manager._enemies
 
+    def get_proximity(self):
```

Comment:
not sure why rooms have a "proximity". I know you added this to retrieve the value in game manager, but it is only the treasure itself that has a proximity. What if we had multiple items in a single room? If the drop logic was put in its own class (as suggested above), it could be added to the room itself.